src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Day/DiaryDayNotesInfoViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+class DiaryDayNotesInfoViewController: DynamicTableViewController, DismissHandling {
+
+	// MARK: - Initializers
+	
+	init() {
+		self.viewModel = DiaryDayNotesInfoViewModel()
+		super.init(nibName: nil, bundle: nil)
+		navigationItem.rightBarButtonItem = dismissHandlingCloseBarButton
+	}
+
+	@available(*, unavailable)
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	// MARK: - View Lifecycle Methods
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+
+		setupTableView()
+	}
+
+	// MARK: - Overrides
+
+	override var navigationItem: UINavigationItem {
+		navigationFooterItem
+	}
+
+	private lazy var navigationFooterItem: ENANavigationFooterItem = {
+		let item = ENANavigationFooterItem()
+
+		item.primaryButtonTitle = AppStrings.NewVersionFeatures.buttonContinue
+		item.isPrimaryButtonEnabled = true
+		item.isSecondaryButtonHidden = true
+
+		return item
+	}()
+	
+	// MARK: - Protocol DismissHandling
+	
+	func wasAttemptedToBeDismissed() {
+		
```

Comment:
Todo?

Reply:
you can remove the whole method, just declare conformance to the protocol is enough